### ISSUE TITLE
New command request - zoom in on gps position

### ISSUE THREAD

**Issue by ollesmaps** [OpenOrienteering/mapper#949]
Please add into the app the possibility to zoom in on current gps position. Zoom around 8,04x (that might be optional). That will make coming back to work faster.

**Comment by ollesmaps** [comment]
It could zoom into defined mapping scale as mentioned in #1005.
If location info is not available it could zoom into place where last editing occured.

**Comment by lpechacek** [comment]
See [pull request 1028](https://github.com/OpenOrienteering/mapper/pull/1028). It does not change the zoom level bot moves the map view so that current position fix is in the centre. After spending about 10 hours in the field so far, testing the new feature, I agree it's a big time saver.

**Comment by ollesmaps** [comment]
Great! I am looking forward to test it.

**Comment by ollesmaps** [comment]
I included the zoom into mapping scale at the same time as fixing map view into current position (in feature request) because I think that is what I would do anyway.  After fixing map view into current position I would like to start immediately mapping and for that I need a zoom into a maping scale. I think that is a workflow - fixing map view into current position, zoom in - for majority of uses. So I combined it in the request. 
For the other cases there are Show whole map, Zoom out buttons.
